tests/unit/test_operator.py

```diff
@@ -31,6 +31,20 @@ def test_log_forwarding(
         mock_logging.assert_called_once_with(charm=harness.charm)
 
 
+def test_metrics(
+    harness,  # noqa F811
+    mocked_kubernetes_service_patcher,  # noqa F811
+    mocked_resource_handler,  # noqa F811
```

Comment:
Is there a chance we need to udpate the linting rules in this repo? I see that we need to define this `noqa` in all of the functions so it could be that the config is different than other repositories

Reply:
Fixed :+1: